polaris-react/src/components/Filters/components/FilterPill/FilterPill.scss

```diff
@@ -26,8 +26,22 @@
     }
   }
 
+  &:hover,
+  &:active {
+    border-style: solid;
+  }
+
+  &:hover,
+  &:focus {
+    background: transparent;
+  }
+
+  &:active {
+    background: var(--p-color-bg-subdued);
+  }
+
   &.disabledFilterButton {
-    background: var(--p-color-bg-disabled);
+    background: var(--p-color-bg-transparent-disabled-experimental);
```

Comment:
changed to match [Figma](https://www.figma.com/file/jLLkmo9r28hiqPvf4s90E4/Polaris-Gen-3-Components-%5Balpha%5D?node-id=44707%3A19512&mode=dev). 

But also, will remove these entirely in a subsequent PR, since [disabled FilterPills are no longer rendered](https://github.com/Shopify/polaris/pull/8876/files#diff-82b9db31501c337b5191fcba9d9e77b9a5875f84b986930a168d6d9801418185)